src/store/modules/gameList.js

```diff
@@ -8,51 +8,75 @@ class GameSummary {
     this.numPlayers = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players.length : 0;
     this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : false;
     this.isRanked = Object.prototype.hasOwnProperty.call(obj, 'isRanked') ? obj.isRanked : false;
+    this.isOver = false;
```

Comment:
Should this be hardcoded here?

Reply:
I think yes. GameSummaries are first instantiated clients side when we request the games from the server (open and spectate) and the server doesn't send finished games, so all of them are initially `isFinished = false` we only flip this to true when a game was not finished during initial page load and then finishes while we're on the home page (which emits a socket event)

Reply:
Man, I must have looked at this a dozen times and completely missed this was the class usage. Ignore me, haha. This is right, I completely missed this is the original instantiating of the variable.